ml_git/ml_git_message.py

```diff
@@ -66,6 +108,74 @@
+    'INFO_CORRUPTED_CANNOT_BE_ADD': 'The following files cannot be added because they are corrupted:',
+    'INFO_CHECKOUT_BARE_MODE': 'Checkout in bare mode done.',
+    'INFO_FILE_ALREADY_EXISTS_REPOSITORY': 'The file %s already exists in the repository. If you commit, the file will be overwritten.',
+    'INFO_MUTABILITY_CANNOT_BE_STRICT': 'You cannot use this command for this entity because mutability cannot be strict.',
+    'INFO_PERMISSIONS_CHANGED_FOR': 'The permissions for %s have been changed.',
+    'INFO_FILE_NOT_FOUND': 'File %s not found',
+    'INFO_TAG': 'Tag: %s',
+    'INFO_MESSAGE': 'Message: %s',
+    'INFO_FILES_TOTAL': 'Total of files: %d',
+    'INFO_WORKSPACE_SIZE': 'Workspace size: %s',
+    'INFO_ADDED_FILES': 'Added files [%s]',
+    'INFO_DELETED_FILES': 'Deleted files [%s]',
+    'INFO_FILES_SIZE': 'Files size: %s',
+    'INFO_AMOUNT_SIZE': 'Amount of files: %s',
+    'INFO_NOT_INITIALIZED': 'The %s doesn\'t have been initialized.',
+    'INFO_NONE_ENTITY_ANAGED': 'You don\'t have any entity being managed.',
+    'INFO_ENTITY_NAME_EXISTS': 'An entity with that name already exists.',
+    'INFO_EXCLUSIVE_IMPORT_ARGUMENT': 'The argument `import` is mutually exclusive with argument',
+    'INFO_EXCLUSIVE_CREDENTIALS_PATH_ARGUMENT': 'The argument `credentials_path` is required if `import-url` is used.',
+    'INFO_INVALID_URL': 'Invalid url: [%s]',
+    'INFO_UNZIPPING_FILES': 'Unzipping files',
+    'INFO_NO_SUCH_OPTION': 'Error: no such option: --sample-type',
+    'INFO_NOT_INITIALIZED_METADATA': 'Could not initialize metadata for %s.',
+    'INFO_CANNOT_ADD_NEW_DATA_AN_ENTITY': 'You cannot add new data to an entity that is based on a checkout with the --sampling option.',
+    'INFO_INITIALIZING_PROJECT': 'Initializing the project with global settings',
+    'INFO_ARE_NOT_IN_INITIALIZED_PROJECT': 'You are not in an initialized ml-git repository and do not have a global configuration.',
+    'INFO_NOT_GIT_REPOSITORY': 'fatal: not a git repository (or any of the parent directories): .git',
+    'INFO_PATHSPEC_KNOWN_GIT': 'error: pathspec \'%s\' did not match any file(s) known to git',
+    'INFO_UNABLE_AZURE_CONNECTION': 'Unable to connect to the Azure storage.',
+    'INFO_ALREADY_EXISTS': 'Tag [%s] already exists in the ml-git repository.',
+    'INFO_COULD_NOT_UPDATE_METADATA': 'Could not update metadata. Check your remote configuration.',
+    'INFO_CREATE_ABORTED': 'Create command aborted!',
+    'INFO_ALREAD_IN_REPOSITORY': 'You already are in a ml-git repository (%s)',
+    'INFO_NO_FILES_COMMIT_FOR': 'No files to commit for [%s]',
+    'INFO_ADDING_PATH_TO': '%s adding path [%s] to ml-git index',
+    'INFO_STATUS_OF': '%s: status of ml-git index for [%s]',
+    'INFO_NO_HEAD_FOR': 'No HEAD for [%s]',
+    'INFO_INITIALIZING_DATASET_DOWNLOAD': 'Initializing related dataset download',
+    'INFO_INITIALIZING_LABELS_DOWNLOAD': 'Initializing related labels download',
```

Comment:
This message should be simplified into one.

```suggestion
    'INFO_INITIALIZING_ENTITY_DOWNLOAD': 'Initializing related %s download',
```

Reply:
Done on commit: [a68fb86](https://github.com/HPInc/ml-git/pull/90/commits/a68fb86503522cbc06d98f096a56656bc439437a)